egs/wsj/s5/utils/adjust_unk_graph.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
please put this in utils/lang/; I'm avoiding putting things in the top level as it's getting crowded.